content/projects/intro-to-database/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Introduction to database
+
+ready: True
+
+## Task
+
+- create a database and name it “mystore”
+- Create tables the following tables :customers, product items, suppliers, orders.
+- Add properties to table (be creative here)
+- Link tables  (relationships)
+- Write a query to populate each table with at least 5 items
+- Write a query to check the amount of product item left
+- write a query to check the amount of orders
```

Comment:
Capitalise the beginning of all or none of the sentences, it looks off if all but 2 are capitalised